src/test-apps/happy/lib/WeaveNetworkGateway.py

```diff
@@ -30,6 +30,7 @@
 
 from happy.ReturnMsg import ReturnMsg
 from happy.Utils import *
+from happy_utils.IP import IP
```

Comment:
We already have happy.Utils. What is the rationale for creating something parallel called happy_utils.IP? Can we not just create happy.Utils.IP as a new subset of happy.Utils?

Reply:
Utils is a python file in happy which is Utils.py, 
happy_utils is a folder to put all utils in Happy.
As first step, I only put IP.py util under happy_utils, I plan to review and move Utils.py under happy_utils folder future.